nextflow.config

```
@@ -12,8 +12,10 @@ params {
     // Input options
     input                      = null
     id                         = 'placement'
-    queryfile                  = 'queries.faa'
-    refalignment               = 'reference.alnfaa'
+    alignmethod                = 'hmmer'
+    queryseqfile               = 'queries.faa'
+    refseqfile                 = 'reference.alnfaa'
+    hmmfile                    = null
     refphylogeny               = 'reference.newick'
```

Can you really have that defaults here? I assume not
```suggestion
    queryseqfile               = null
    refseqfile                 = null
    hmmfile                    = null
    refphylogeny               = null
```

---

You're right, probably better not to have.